File: src/nomanssave/eK.java
package nomanssave;

import java.util.Comparator;

class eK implements Comparator {
   public int a(eI var1, eI var2) {
      return var1.name.compareTo(var2.name);
   }

   // $FF: synthetic method
   public int compare(Object var1, Object var2) {
      return this.a((eI)var1, (eI)var2);
   }
}

File: NMSSaveEditor/nomanssave/mixed/eK.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace NMSSaveEditor
{

public class eK : IComparer<object> {
   public int a(eI var1, eI var2) {
      return var1.name.CompareTo(var2.name);
   }
   public int Compare(object var1, object var2) {
      return this.a((eI)var1, (eI)var2);
   }
}

}
